### PR TITLE
Docs: Update Piral Inspector and Piral VS Code

### PR DESCRIPTION
**Pull request by manuelroemer**
# New Pull Request

For more information, see the `CONTRIBUTING` guide.

## Prerequisites

Please make sure you can check the following boxes:

- [x] I have read the **CONTRIBUTING** document
- [x] My code follows the code style of this project
- [x] All new and existing tests passed

### Contribution Type

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [ ] Bug fix (non-breaking change which fixes an issue, please reference the issue id)
- [ ] New feature (non-breaking change which adds functionality, make sure to open an associated issue first)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes

### Description

This includes some updates/refreshments to the Piral Inspector and Piral VS Code documentation. This addresses #541 and #540.

### CODE REVIEW

File: docs/tools/piral-inspector.md [hunk 1]
```diff
@@ -2,229 +2,199 @@
...
+This feature only has an impact when debugging _pilets_, i.e., when `pilet debug` is running.
 :::
 
-![Add pilets to the Piral Inspector](../diagrams/piralinspector_tool_addpilets.png)
-
-### Registered Routes
+Refreshes the current browser page when the currently debugged pilet changes.
 
-This section shows all currently registered routes, and with a click on a single one, it will call the route.
+### Visualize component origins
```
Comment (FlorianRappl) [inline docs/tools/piral-inspector.md:179]: Can we keep title case?

Above we have things like "State Container Logging" but then we have "Full refresh on change" or "Visualize component origins". Would be great to use instead "Full Refresh on Change" or "Visualize Component Origins" etc.

File: docs/tools/piral-inspector.md [hunk 2]
```diff
@@ -2,229 +2,199 @@
...
 
-![Registered routes in the Piral Inspector](../diagrams/piralinspector_tool_registeredroutes.png)
+While enabled, provides an additional button that visually indicates the _origin_ of the page's components, i.e. the pilet where a component is registered. This can be very useful for analyzing Piral applications, e.g. for detecting which pilet needs to be updated when you want to make changes to a specific component.
```
Comment (FlorianRappl) [inline docs/tools/piral-inspector.md:181]: Not sure what we currently use - I think we always use commas after "i.e.," and "e.g.,", but maybe we changed this. Can you have a look what is consistent and either keep or adjust this here? Thanks!

File: docs/tools/vscode-piral.md [hunk 3]
```diff
@@ -1,23 +1,69 @@
...
 
-Piral comes with a Visual Studio Code extension that's available at the official [Visual Studio Code Marketplace](https://marketplace.visualstudio.com/items?itemName=smapiot.vscode-piral).
+## Introduction
+
+[Piral VS Code](https://github.com/smapiot/vscode-piral) is an extension for [Visual Studio Code](https://code.visualstudio.com/). It provides many utilities for developing Piral applications, directly integrated into the editor.
+
+You can install the extension from:
+
+[![VS Code Logo](../diagrams/vscode-vscode-logo.svg){style="background-color: transparent"} **VS Code Marketplace**](https://marketplace.visualstudio.com/items?itemName=smapiot.vscode-piral) &nbsp; &middot; &nbsp;
+[![VSX Logo](../diagrams/vscode-vsx-logo.svg){style="background-color: transparent"} **Open VSX**](https://open-vsx.org/extension/smapiot/vscode-piral)
```
Comment (FlorianRappl) [inline docs/tools/vscode-piral.md:14]: Maybe add a sentence that the Open VSX marketplace is used in VS Code Compatible Editors such as Gitpod, Eclipse Theia, or VSCodium.

**Review by FlorianRappl — COMMENTED** [review]
Looks fantastic!

Thanks for the outstanding work 🚀

File: docs/tools/piral-inspector.md (hunk 1 above)
Comment (manuelroemer) [reply docs/tools/piral-inspector.md:179]: I can change that, but in this specific case I did it on purpose (I copied the name of the setting 1:1).

**Review by FlorianRappl — APPROVED** [review]

**Pull request merged**